pkg/config/controller_config.go

```diff
@@ -141,3 +159,13 @@ func (cfg *ControllerConfig) validateExternalManagedTagsCollisionWithDefaultTags
 	}
 	return nil
 }
+
+func (cfg *ControllerConfig) validateBackendSecurityGroupConfiguration() error {
+	if len(cfg.BackendSecurityGroup) == 0 {
+		return nil
+	}
+	if strings.HasPrefix(cfg.BackendSecurityGroup, "sg-") {
```

Comment:
should this be !strings.HasPrefix(cfg.BackendSecurityGroup, "sg-")

Reply:
Yes. Thanks.